Cargo.toml

```diff
@@ -4,6 +4,13 @@ version = "0.1.0"
 authors = ["mlange-42 <martin_lange_@gmx.net>"]
 edition = "2018"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[[bin]]
```

Comment:
I would have expected both sections to be automatically detected?

Reply:
Ah, indeed...